src/main/java/reposense/RepoSense.java

```diff
@@ -23,8 +24,11 @@ public static void main(String[] args) {
             List<RepoConfiguration> configs = CsvParser.parse(cliArguments.getConfigFilePath());
             RepoConfiguration.setDatesToRepoConfigs(configs, cliArguments.getSinceDate(), cliArguments.getUntilDate());
 
-            ReportGenerator.generateReposReport(
+            String generatedFolderName = ReportGenerator.generateReposReport(
                     configs, cliArguments.getOutputFilePath().toAbsolutePath().toString());
+
+            FileUtil.zipJson(cliArguments.getOutputFilePath().toAbsolutePath().toString(), generatedFolderName);
```

Comment:
Should we put this into `ReportGenerator#generateReposReport(...)` instead, that we can do a test for the `.zip` file as well in functional test?

Reply:
But is zipping the jsons a part of generating the report?

Reply:
>But is zipping the jsons a part of generating the report?

Hmmm, not too sure on that part, let's ignore this for now.